### PR TITLE
Reviewing Lines Part 2

### PR DESCRIPTION
**Pull request by ttgmichael**
Dolar signs on QUESTLV100 lines 700-1000 with some revisions.

Let me know your thoughts! :)

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1]: Extra line? Needs to be removed.

**Comment by Sunaries** [comment]
Dumbla Pond to Dumbal Pond - Was this also changed in everything else?

**Comment by Sunaries** [comment]
What is the meaning of Dumbliai? The wiki does say 'dumbla' in the second column.

**Comment by ttgmichael** [comment]
Dumbliai is the correct form of algae, I'm not sure if you can just shorten it to Dumbla https://lt.wikipedia.org/wiki/Dumbliai

https://translate.google.com/?ie=UTF-8&hl=en&client=tw-ob#en/lt/algae

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**